### ISSUE TITLE
testing and validation sets

### ISSUE THREAD

**Issue by rudylyh** [Xiangyi1996/PPNet-PyTorch#10]
May I ask which validation data are you using to select the best model during training? It seems to be the same with the testing data in the code. Thank you!

**Comment by Xiangyi1996** [comment]
We randomly sample a set of tasks (1000) from D^{te} as the validation set.  But at test state, we randomly sample another 5000 tasks from D^{te}.
